examples/run/sycl/full_chain_algorithm.sycl

```diff
@@ -103,8 +103,6 @@ full_chain_algorithm::full_chain_algorithm(const full_chain_algorithm& parent)
 
 full_chain_algorithm::~full_chain_algorithm() {
     // Need to ensure that objects would be deleted in the correct order.
-    m_cached_device_mr.reset();
-    m_device_mr.reset();
```

Comment:
You might as well remove the comment. :thinking:

Reply:
Notice that there is still a deletion under this comment (which might also be a bug, actually; goes to show how bad of an idea it usually is to define custom destructors); so this comment should stay.